### DISCUSSION TITLE
Outdated links in readme lead to hijacked repo

### DISCUSSION THREAD

**Discussion by AaronLi** [stm32duino/Arduino_Core_STM32#1927] [General]
Under [Generic STM32F4 boards](https://github.com/stm32duino/Arduino_Core_STM32#generic-stm32f4-boards) the More Info link for the STM32F401CC and STM32F411CE WeAct Black Pill boards lead to this hijacked repo https://github.com/cmicot/MiniF4-STM32F4x1 when they should probably lead to https://github.com/WeActStudio/WeActStudio.MiniSTM32F4x1

**Comment by fpistm** [comment]
Right,
this link
https://github.com/WeActTC/MiniF4-STM32F4x1
was redirected to the one you gave.

Fixed. Thanks for the report

https://github.com/stm32duino/Arduino_Core_STM32/commit/4b5098cfb8eb9d06bb1c55ad74f04dc2f042fdcf
